GetGroup/index.ts

```diff
@@ -0,0 +1,33 @@
+import { AzureFunction, Context, HttpRequest } from "@azure/functions";
+import Group from "../util/group.model";
+import IPerson from "../util/person.interface";
+import { authWrapper, userPerm } from "../util/authorization";
+import { noPermissionError, notFoundError } from "../util/custom.error";
+import errorHandler from "../util/error.handling";
+import mongooseConnection from "../util/mongoose.connection";
+import { validateId } from "../util/joi";
+
+const GetGroup: AzureFunction = async function (
+  context: Context,
+  _req: HttpRequest,
+  user: IPerson
+): Promise<void> {
+  try {
+    const { id } = context.bindingData;
+    validateId(id);
+
+    const group = await Group.findById(id).exec();
+
+    if (!group) throw new notFoundError("group");
+    if (!group.people.includes(user.id)) throw new noPermissionError("an admin");
```

Comment:
change it to something more readable
```suggestion
    if (!group.people.includes(user.id)) throw new noPermissionError("User has no permission to get a group");
```